ts/components/conversation/TimelineMessage.tsx

```diff
@@ -212,22 +213,36 @@ export function TimelineMessage(props: Props): JSX.Element {
         event.stopPropagation();
       }
 
-      if (!attachments || attachments.length !== 1) {
+      if (!attachments) {
         return;
       }
 
+      // check if any attachment needs to be downloaded from servers
       const attachment = attachments[0];
-      if (!isDownloaded(attachment)) {
-        kickOffAttachmentDownload({
-          attachment,
-          messageId: id,
-        });
-        return;
+      for (const attachment of attachments) {
```

Comment:
This `const attachment` is shadowing the previous line. We probably don't need that anyway, just directly use `attachments[0]` in the `saveAttachment()` below.